Doc/library/shutil.rst

```diff
@@ -680,6 +681,33 @@ The resulting archive contains:
     -rw-r--r-- tarek/staff   37192 2010-02-06 18:23:10 ./known_hosts
 
 
+.. _shutil-archiving-example-with-basedir:
+
+Archiving example with *base_dir*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, we have a file :file:`please_add.txt` within the directory
+structure :file:`tmp/root/structure/content`, which we want to add to the
+archive along with the directories :file:`structure` and :file:`content`, and a
+second file :file:`do_not_add.txt` in :file:`tmp/root/structure`, which we don't
+want to have in the archive. Therefore we use the following::
+
+    >>> from shutil import make_archive
+    >>> make_archive(base_name='tmp/archive',
+                     format='tar',
+                     root_dir='tmp/root',
+                     base_dir='structure/content/')
+    '/Users/user/tmp/archive.tar'
+
+Listing the files in the resulting archive gives us the following:
+
+.. code-block:: shell-session
+
+    $ tar -tvf tmp/archive.zip
```

Comment:
I'd rather use `python -m tarfile -l archive.tar` here :)

(Also, there's a typo in the file name: .zip -> .tar)